gammapy/image/tests/test_profile.py

```diff
@@ -83,18 +86,47 @@ def test_image_lon_profile():
     lon_profile1 = image_profile('lon', image, lat, lon, binsz,
                                  errors=True)
     # atol 0.1 is sufficient to check if correct number of pixels are included
-    assert_allclose(lon_profile1['BIN_VALUE'].data.astype(float),
+    assert_allclose(lon_profile1.table['profile'].data.astype(float),
                     1000 * np.ones(79), rtol=1, atol=0.1)
-    assert_allclose(lon_profile1['BIN_ERR'].data,
-                    0.1 * lon_profile1['BIN_VALUE'].data)
+    assert_allclose(lon_profile1.table['profile_err'].data,
+                    0.1 * lon_profile1.table['profile'].data)
 
     lon_profile2 = image_profile('lon', image, lat, lon, binsz,
                                  counts, errors=True)
     # atol 0.1 is sufficient to check if correct number of pixels are included
-    assert_allclose(lon_profile2['BIN_ERR'].data,
+    assert_allclose(lon_profile2.table['profile_err'].data,
                     31.622776601683793 * np.ones(79), rtol=1, atol=0.1)
 
     lon_profile3 = image_profile('lon', image, lat, lon, binsz, counts,
                                  mask_array, errors=True)
 
-    assert_allclose(lon_profile3['BIN_VALUE'].data, np.zeros(79))
+    assert_allclose(lon_profile3.table['profile'].data, np.zeros(79))
+
+
+class TestImageProfile(object):
+    def setup(self):
+        table = Table()
+        table['x_ref'] = np.arange(-90, 90, 10) * u.deg
+        table['profile'] = np.cos(table['x_ref'].to('rad')) * u.Unit('cm-2 s-1')
+        table['profile_err'] = 0.1 * table['profile']
+        self.profile = ImageProfile(table)
+
+    def test_normalize(self):
+        normalized = self.profile.normalize(mode='integral')
+        profile = normalized.profile
+        assert_quantity_allclose(profile.sum(), 1 * u.Unit('cm-2 s-1'))
+
+        normalized = self.profile.normalize(mode='peak')
+        profile = normalized.profile
+        assert_quantity_allclose(profile.max(), 1 * u.Unit('cm-2 s-1'))
+
+
+    @requires_dependency('scipy')
+    def test_smooth(self):
```

Comment:
Add some more tests and asserts for smooth?
E.g. Gauss smooth is never executed I think -> does it work?
Does `profile_err` get filled with reasonable values?